plugins/modules/zos_job_query.py

```diff
@@ -237,6 +237,10 @@
         Returned when Z Open Automation Utilities (ZOAU) is 1.2.4 or later.
       type: str
       sample: "IEBGENER"
+    execution_time:
+      description: The time the JCL executed for.
```

Comment:
I think we can be a bit more verbose about this option, pending to check how this behaves with long running jobs.
```suggestion
      description: Total duration time of the job execution, if it has finished. If the job is still running, it represents the time elapsed from the job execution start and current time.
```